index.html

```diff
@@ -87,131 +113,51 @@
           <li><button value="9">9</button></li>
           <li><button value="0">0</button></li>
           <li><button value=".">.</button></li>
-          <li><button data-action="plus">+</button></li>
-          <li><button data-action="minus">-</button></li>
-          <li><button data-action="multiply">*</button></li>
-          <li><button data-action="divide">/</button></li>         
-          <li><button data-action="clear">c</button></li>
-          <li><button data-action="memoryc">MC</button></li>
-          <li><button data-action="memoryr">MR</button></li>
-          <li><button data-action="memorys">MS</button></li>
-          <li><button data-action="equal">=</button></li>
+          
         </ul>        
 
 <script type="text/javascript">
-  var result;
-  var tempPlus;
+  var result = "";
```

Comment:
There is too much code in one place so it is high to you created separate files for everything